src/api/app/helpers/webui/notification_excerpt_helper.rb

```diff
@@ -1,28 +1,23 @@
-class NotificationExcerptComponent < ApplicationComponent
+module Webui::NotificationExcerptHelper
   TRUNCATION_LENGTH = 100
   TRUNCATION_ELLIPSIS_LENGTH = 3 # `...` is the default ellipsis for String#truncate
 
-  def initialize(notifiable)
-    super
-
-    @notifiable = notifiable
-  end
-
-  def call
-    text = case @notifiable.class.name
+  # TODO: Content of ViewComponent. Move to sub-classes once STI is set.
+  def excerpt(notification)
```

Comment:
I think it would have been better to pass `notifiable` instead of `notification`.

Reply:
See https://github.com/openSUSE/open-build-service/pull/16341